libember_slim/Source/ber.h

```diff
@@ -20,6 +20,12 @@
 #ifndef __LIBEMBER_SLIM_BER_H
 #define __LIBEMBER_SLIM_BER_H
 
+#ifdef LIBEMBER_DLL_EXPORTS
+#define LIBRARY_API __declspec(dllexport)

```

Comment:
I'm not a C expert, but I get the point, and I think I will be able to do this. Is there any recommended name for the header file? Should it be "api.h", just like the api.hpp file from the C++ library?